README.md

```diff
@@ -2,19 +2,19 @@
 
-**Warning:** This specification is currently in 1.0 'beta' releases, which means we don't anticipate anything major will change, but still reserve the right to make changes in backwards incompatible ways. See the [versioning](#versioning) section below for more info. If you are excited about the potential please collaborate with us by building implementations, sounding in on the issues and contributing PR's! Note that the specification in this repository is in flux, we recommend using the [latest published release](https://geoparquet.org/releases/).
+**Note:** This specification is currently in 1.0 'release candidate' status, which means the community is proposing the current version to be 1.0.0, and if no blocking negative feedback is made in the next few weeks (as of late July) then it will become 1.0.0. This means breaking changes are still possible, but quite unlikely - see the [versioning](#versioning) section below for more info.
 
 Early contributors include developers from GeoPandas, GeoTrellis, OpenLayers, Vis.gl, Voltron Data, Microsoft, Carto, Azavea, Planet & Unfolded.
-Anyone is welcome to join us, by building implementations, trying it out, giving feedback through issues and contributing to the spec via pull requests.
-Initial work started in the [geo-arrow-spec](https://github.com/geopandas/geo-arrow-spec/) GeoPandas repository, and that will continue on
-Arrow work in a compatible way, with this specification focused solely on Parquet.
+Anyone is welcome to join the project, by building implementations, trying it out, giving feedback through issues and contributing to the spec via pull requests.
+Initial work started in the [geo-arrow-spec](https://github.com/geoarrow/geoarrow) GeoPandas repository, and that will continue on
+Arrow work in a compatible way, with this specification focused solely on Parquet. We are in the process of becoming an [OGC](https://ogc.org) official
+[Standards Working Group](https://portal.ogc.org/files/103450) and are on the path to be a full OGC standard.
 
 - [**Specification**](format-specs/geoparquet.md)
 - [JSON Schema](format-specs/schema.json)
 - [Examples](examples/)
-- [Validator](validator/)
```

Comment:
The link is getting removed but the folder is still in the repo. That feels kinda weird.

Reply:
Yeah, we opened https://github.com/geoparquet

It's on my list to move it out and over, but didn't want to block rc.1 on that.